docker/server/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && apt-get -y install python-pip software-properties-common \
+ca-certificates apt-transport-https apt-utils \
+libterm-readline-gnu-perl software-properties-common \
+wget curl sudo
```

Comment:
```suggestion
RUN apt-get update && apt-get -y install python-pip \
  software-properties-common \
  ca-certificates \
  apt-transport-https \
  apt-utils \
  libterm-readline-gnu-perl \
  software-properties-common \
  wget \
  curl \
  sudo \
```

Reply:
Do you need both wget and curl?

Reply:
It's also usually a good idea to remove any artifacts from apt, to make the image smaller:
Ex:

```
RUN apt-get update && apt-get install -y --no-install-recommends \
    software-properties-common \
    ... \
  && rm -rf /var/lib/apt/lists/*
```

Reply:
* removed wget/curl (used it before, forgot to remove them)
* ordered packaged in alphabetical order for readability